.circleci/integration-tests/Dockerfile

```diff
@@ -71,6 +107,22 @@ RUN curl -o kubectl https://s3.us-west-2.amazonaws.com/amazon-eks/1.22.6/2022-03
     && chmod +x ./kubectl \
     && mv ./kubectl /usr/local/bin
 
+#install Apache Hive 2.3.9 (Download same version of library to match EMR hive version created by DAG)
+RUN curl "https://downloads.apache.org/hive/hive-2.3.9/apache-hive-2.3.9-bin.tar.gz" -o  "/tmp/apache-hive-2.3.9-bin.tar.gz"  \
```

Comment:
consider reading the hive and hadoop versions from environment variables than hard coding the version

Reply:
Done added ENV variable